README.md

```diff
@@ -121,6 +121,23 @@ DATABASES = {
 `OPTIONS` is an optional dictionary of parameters that will be passed to
 [`MongoClient`](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html).
 
+Alternatively, if you prefer to simply paste in a MongoDB URI rather than
+parsing it into the format above, you can use:
+
+```python
+import django_mongodb
+
+MONGODB_URI = "mongodb+srv://myDatabaseUser:D1fficultP%40ssw0rd@cluster0.example.mongodb.net/myDatabase?retryWrites=true&w=majority&tls=false"
+DATABASES["default"] = django_mongodb.parse_uri(MONGODB_URI)
+```
+
+#### `django_mongodb.parse_uri(uri, conn_max_age=0, conn_health_checks=False, test=None)`
+
+- Use `conn_max_age` and `conn_health_checks` to configure [persistent database
+  connections](https://docs.djangoproject.com/en/stable/ref/databases/#persistent-database-connections).
+- Use `test` to provide a dictionary of [settings for test databases](
```

Comment:
Errr, 4d155ac084f13246374142b70c8d429230471173

Reply:
Alex, your first intuition is correct. These variables control Django behavior, not the database driver.

Persistent connections are supported:
https://github.com/django/django/blob/ded485464214a3f69b64402b7d82221279f80008/django/db/backends/base/base.py#L249-L250
https://github.com/django/django/blob/ded485464214a3f69b64402b7d82221279f80008/django/db/backends/base/base.py#L615-L617

Health check don't work because we don't use the cursor API: 
https://github.com/django/django/blob/ded485464214a3f69b64402b7d82221279f80008/django/db/backends/base/base.py#L248
(`backends.base.test_base.ConnectionHealthChecksTests` is skipped)

Reply:
0501907c71ed4a1b1ff74a04bb380d20d3cb6784